incubator/hnc/api/v1alpha2/hnc_config_conversion.go

```diff
@@ -0,0 +1,191 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"strings"
+
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+
+	v1a1 "sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+const (
+	unknownv1a1Prefix = "UnknownV1Alpha1Value-"
+	unknownv1a2Prefix = "UnknownV1Alpha2Value-"
+)
+
+var (
+	// Maps to and from the Hub version (v1alpha1) for all valid values.
+	toMap = map[SynchronizationMode]v1a1.SynchronizationMode{
```

Comment:
Nit: the "to" and "from" here are ambiguous, maybe something like `toV1A1` and `toV1A2` would be more clear?

Including the name of the type ("map") is sometimes a code smell - describe what the variable is _for_, not how it's implemented.

Reply:
sgtm. updated!